### PR TITLE
Update mission list for mobile

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: components/Mission/InviteeGridList.vue [hunk 1]
```diff
@@ -52,6 +40,9 @@ export default {
       type: Array,
       default: () => [],
     },
+    swiperId: {
+
+    },
```
Comment (williamchong007) [inline components/Mission/InviteeGridList.vue]: `swiperId` 應該至少有 type？

File: components/Mission/InviteeGridList.vue [hunk 2]
```diff
@@ -52,6 +40,9 @@ export default {
       type: Array,
       default: () => [],
     },
+    swiperId: {
+
```
Comment (mileswong) [inline components/Mission/InviteeGridList.vue]: type: String ?

Comment (nwingt) [reply components/Mission/InviteeGridList.vue]: Updated

File: components/Mission/InviteeGridList.vue (hunk 1 above)
Comment (nwingt) [reply components/Mission/InviteeGridList.vue]: Updated

File: components/Mission/List.vue [hunk 3]
```diff
@@ -141,6 +178,11 @@ export default {
         }));
       }
     },
+    onClickSlide(i) {
+      if (this.missions.length > 0) {
+        this.onClick(this.missions[i]);
```
Comment (williamchong007) [inline components/Mission/List.vue]: check 埋 `i < this.missions.length` ?

Comment (nwingt) [reply components/Mission/List.vue]: Updated

**Pull request merged**